File: Code_Coach/python/Security.py
x=input()
g=None
risk=None
l=len(x)
for i in range(l):
    if risk==True:
        if x[i]=='G':
            g=True
        elif x[i]=='$' or x[i]=='T':
            break
    elif x[i]=='$' or x[i]=='T':
        risk =True
if g==True:
    print("quiet")
else:
    print("ALARM")

File: Code_Coach/java/Security.java
import java.util.Scanner;

public class Security {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        String input = in.nextLine();
        Boolean danger = false;
        char[] objs = input.toCharArray();
        for(char x : objs){
            if(x == '$' || x == 'T'){
                if(danger){
                    System.out.println("ALARM");
                    break;
                }
                else
                    danger = !danger;
            }
            if(x == 'G' && danger){
                System.out.println("quiet");
                break;
            }
        }
    }
}